### ISSUE TITLE
Throws error due to getmac on illumos and FreeBSD

### ISSUE THREAD

**Issue by sjorge** [OpenZWave/node-red-contrib-openzwave#66]
Looks like getmac does not work on illumos and FreeBSD (presumably other BSD's too), it would be nice to have the option of setting a UUID manually somehow.

**Comment by sjorge** [comment]
Not ideal but this does make it work.

```
From 66ed148c68caf77663b127a5afd3fb1437b09215 Mon Sep 17 00:00:00 2001
From: sjorge <sjorge@blackdot.be>
Date: Sat, 2 Mar 2019 00:27:01 +0100
Subject: [PATCH] Fallback to hostname as getmac is broken on SmartOS

---
 10-zwave.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/10-zwave.js b/10-zwave.js
index b3e7bfb..74061c5 100644
--- a/10-zwave.js
+++ b/10-zwave.js
@@ -28,7 +28,7 @@ var ozwsharedpackage = JSON.parse(fs.readFileSync(ozwsharedpath+"/package.json")
 var thispackage = JSON.parse(fs.readFileSync(__dirname+'/package.json'));
 
 require('getmac').getMac(function(err, macAddress) {
-  if (err) throw err;
+  if (err) macAddress = require("os").hostname();
   UUIDPREFIX = macAddress.replace(/:/gi, '');
 });
 ```

This does a fallback to the hostname if the macadress cannot be parsed.

**Comment by ekarak** [comment]
resolved with release 1.6.2
